### PR TITLE
ovirt_disk: fix transfer ending

### PR DESCRIPTION
**Pull request by mnecas**
Issue: after uploading the image the status of the disk was OK so it tried to attach the disk to the VM but at that moment it failed because of the unfinished transfer status.

Now we wait longer after the transfer is done and it is removed.

I think it would be best to fix this in the SDK and this could be just a temporary fix.

### CODE REVIEW

**Comment by mnecas** [comment]
@mwperina @nirs

**Comment by mwperina** [comment]
SDK is generated automatically, why should we fixed something here? If something needs to be fixed, then it should be fixed in RESTAPI and/or engine backend no?

@barpavel

**Review by mwperina — CHANGES_REQUESTED** [review]
-1 until we finalize discussion in https://bugzilla.redhat.com/show_bug.cgi?id=1883949

File: plugins/modules/ovirt_disk.py [hunk 1]
```diff
@@ -459,9 +460,13 @@ def transfer(connection, module, direction, transfer_func):
         while transfer.phase in [
             otypes.ImageTransferPhase.TRANSFERRING,
             otypes.ImageTransferPhase.FINALIZING_SUCCESS,
+            otypes.ImageTransferPhase.FINISHED_SUCCESS
         ]:
             time.sleep(module.params['poll_interval'])
-            transfer = transfer_service.get()
+            try:
```
Comment (barpavel) [inline plugins/modules/ovirt_disk.py:466]: Looks like the code is indented to the right 1 tab too much?
Lines:
465-469
476-478

**Comment by barpavel** [comment]
> SDK is generated automatically, why should we fixed something here? If something needs to be fixed, then it should be fixed in RESTAPI and/or engine backend no?

Sounds reasonable. But I prefer to hear @nirs opinion as well.

**Comment by mnecas** [comment]
We have decided to wait for a fix from the storage team in https://bugzilla.redhat.com/show_bug.cgi?id=1849861

**Pull request closed without merging**